app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def comment_count(post_id)
+    counts = Comment.where(post_id: post_id).count
+    post = Post.find(post_id)
+    post.update(comments_counter: counts)
+  end
```

Comment:
- [ ] the definition doesn't need an argument and also we can use just post.increment!(:comment_counter) to update the counter.